From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Aikar <aikar@aikar.co>
Date: Thu, 18 Jun 2015 21:56:03 -0400
Subject: [PATCH] Slow down villager pickup of thrown items

Helps 1.8 Farms let hoppers pick it up before Villager
due to our hopper changes

diff --git a/src/main/java/net/minecraft/world/entity/ai/behavior/BehaviorUtils.java b/src/main/java/net/minecraft/world/entity/ai/behavior/BehaviorUtils.java
index e51a78e52969ec687956d16566494fc524e50cb8..d667c1f29322c19d615c12ed6d86ef254bb220d0 100644
--- a/src/main/java/net/minecraft/world/entity/ai/behavior/BehaviorUtils.java
+++ b/src/main/java/net/minecraft/world/entity/ai/behavior/BehaviorUtils.java
@@ -94,7 +94,8 @@ public class BehaviorUtils {
 
         vec3d1 = vec3d1.normalize().scale(0.30000001192092896D);
         entityitem.setDeltaMovement(vec3d1);
-        entityitem.setDefaultPickUpDelay();
+        //entityitem.defaultPickupDelay(); // EMC
+        entityitem.pickupDelay = 100; // EMC
         // CraftBukkit start
         org.bukkit.event.entity.EntityDropItemEvent event = new org.bukkit.event.entity.EntityDropItemEvent(entity.getBukkitEntity(), (org.bukkit.entity.Item) entityitem.getBukkitEntity());
         entityitem.level.getCraftServer().getPluginManager().callEvent(event);
